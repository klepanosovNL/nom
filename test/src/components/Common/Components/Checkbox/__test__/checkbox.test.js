import React from 'react';
import renderer from 'react-test-renderer';
import { Checkbox } from '../Checkbox';

describe('Checkbox"s tests', () => {
	it('Sent only onChange event', () => {
		const tree = renderer
			.create(<Checkbox onChange={() => {}}></Checkbox>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});

	it('Sent nothing', () => {
		const tree = renderer.create(<Checkbox></Checkbox>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
