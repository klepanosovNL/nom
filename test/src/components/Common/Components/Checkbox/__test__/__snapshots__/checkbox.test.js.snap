// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox"s tests Sent nothing 1`] = `
<input
  type="checkbox"
/>
`;

exports[`Checkbox"s tests Sent only onChange event 1`] = `
<input
  onChange={[Function]}
  type="checkbox"
/>
`;
